specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1459 @@
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/AzurePrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateCloud_GetById": {
+            "$ref": "./examples/PrivateCloud_GetById.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds": {
+      "get": {
+        "operationId": "PrivateCloud_ListInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "list private clouds in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/AzurePrivateCloudResponse"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateCloud_ListInResourceGroup": {
+            "$ref": "./examples/PrivateCloud_ListInResourceGroup.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}": {
+      "get": {
+        "operationId": "PrivateCloud_GetByIdInResourceGroup",
+        "tags": [
+          "PrivateClouds"
+        ],
+        "summary": "get a private cloud in a resource group",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/AzurePrivateCloudResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "PrivateCloud_GetByIdInResourceGroup": {
+            "$ref": "./examples/PrivateCloud_GetByIdInResourceGroup.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "PrivateCloud_Create",
```

Comment:
Can this only create a `PrivateCloud` or also update an existing one?  If it's the latter the name should be `PrivateClouds_CreateOrUpdate`.